lib/httpMock.js

```diff
@@ -15,15 +15,13 @@ function mockTemplate() {
 
         $provide.decorator('$http', ['$delegate', '$q', '$injector', function($http, $q, $injector) {
 
-        var interceptors = $httpProvider.interceptors;
+        var interceptors = [];
 
-        function getInterceptor(interceptorExpression) {
-            if (angular.isString(interceptorExpression)) {
-                return $injector.get(interceptorExpression);
-            } else {
-                return $injector.invoke(interceptorExpression);
-            }
-        }
+        angular.forEach($httpProvider.interceptors, function(interceptorExpression) {
```

Comment:
I'm not sure if it happens, but if interceptors are added to `$httpProvider.interceptors` after this loop runs then they'll be ignored.